README.md

```diff
@@ -36,20 +36,31 @@ This module gives you the possibility to integrate OneSignal into you're Appcele
 ### Usage
 1. Register device for Push Notifications
    
-   ```
+   ```js
    	    // This registers your device automatically into OneSignal
        var onesignal = require('com.williamrijksen.onesignal');
    ```
 1. To add the possibility to target people for notifications, send a tag:
    
-   ```
+   ```js
        onesignal.sendTag({ key: 'foo', value: 'bar' });
-   ```
+   ```js
```

Comment:
Because it looks like this

![schermafbeelding 2016-11-10 om 13 07 21](https://cloud.githubusercontent.com/assets/1991582/20176145/a9a724be-a746-11e6-99ec-189b9ab4717d.png)